### ISSUE TITLE
Connect to Multiple LED Lamp

### ISSUE THREAD

**Issue by leedashone** [kentya6/NexturnController#1]
Hi!  Thanks for your great code.  
I have many BLE RGB LED lamps.,
Is it possible control multiple lamps at same time?
Many thanks!

**Comment by kentya6** [comment]
Hi leedashone!

It is possible to control multiple BLE LED lamps at same time.
To make it realized, send LED data for each lamps at same time.

I will commit some code able to controlling multiple LED lamps (named "Nexturn") within 1-2 week.
Please wait for it. :)

**Comment by leedashone** [comment]
BIG Thanks!  I will happy waiting for your new code  ;-)

And, I found sometimes CoreBluetooth's didDiscoverPeripheral  & didConnectPeripheral 
function will call-back without peripheral name. (ps: name == nil)
So the "switch peripheral.name"  instruction may take xCode error (or app force close)
We may add " If peripheral.name == nil { return }  "  to let our code safe.

**Comment by kentya6** [comment]
Thank you for waiting!

I wrote some code to support multiple Nexturn devices at the same time. (see https://github.com/kentya6/NexturnController/commit/c337030a16753d47f40bd839512f9d96f49618d4)
To support multiple Nexturn devices, I deleted CoreBluetooth method, named `stopScan()`.
Battery consumption will increase, but automatically add the new Nexturn, and can be controlled at the same time.

And thank you for pointing out related to peripheral name.
I fixed it. :) (see https://github.com/kentya6/NexturnController/commit/c347aba1a4b993facde17d13d596cc2765108643)

Thanks.
